templates/zerver/help/include/event-filtering-instruction.md

```diff
@@ -0,0 +1,12 @@
+(Optional) To filter the events that trigger the notifications, you can append
```

Comment:
I would remove "(Optional)" as the previous two paragraphs are also describing optional updates to the URL.